cmd/lk/token.go

```diff
@@ -266,7 +279,61 @@ func createToken(ctx context.Context, c *cli.Command) error {
 	}
 
 	if !hasPerms {
-		return errors.New("no permissions were given in this grant, see --help")
+		type permission uint
+
+		const (
+			pCreate permission = iota
+			pList
+			pJoin
+			pAdmin
+			pRecord
+			pEgress
+			pIngress
+			pMetadata
+		)
+
+		permissions := make([]permission, 0)
+
+		if err := huh.NewMultiSelect[permission]().
+			Options(
+				huh.NewOption("Create", pCreate),
+				huh.NewOption("List", pList),
+				huh.NewOption("Join", pJoin),
+				huh.NewOption("Admin", pAdmin),
+				huh.NewOption("Record", pRecord),
```

Comment:
we don't want to issue `Record` permissions. for context, Egress instance will create their own tokens and join the room setting the `recorder` token and `participant.kind=egress`

Reply:
Ah ok. So remove this flag altogether?

Reply:
yeah. that'd be great!